components/Layout.tsx

```diff
@@ -36,7 +36,7 @@ const Layout = ({
 				{/*<!--Tracking Code End of HubSpot Embed Code -->*/}
 			</Head>
 			<NextSeo
-				title={`APIClarity | ${pageTitle}`}
+				title={`OpenClarity | ${pageTitle}`}
 				description="Open source for API traffic visibility in K8s clusters"
 				canonical={`https://apiclarity.io${pagePath || ""}`}
```

Comment:
Might want to remove APIclarity reference here.

Reply:
I believe this is fixed in 6ed1a5c287472f9917ff1c9dc345ecaab15636d1

Reply:
LGTM.